FirebaseRemoteConfig/Sources/Private/RCNConfigFetch.h

```diff
@@ -53,6 +53,8 @@ typedef void (^RCNConfigFetcherCompletion)(NSData *data, NSURLResponse *response
 /// Add the ability to update NSURLSession's timeout after a session has already been created.
 - (void)recreateNetworkSession;
 
+- (NSString *)getTemplateVersionNumber;
```

Comment:
Better define this as a ivar _templateVersionNumber internally and here as a property.

Then internally use getTemplateVersionNumber to update the ivar _templateVersionNumber